lib/paginator.ex

```diff
@@ -24,7 +24,8 @@ defmodule Paginator do
         use Paginator,
           limit: 10,                  # sets the default limit to 10
           maximum_limit: 100,         # sets the maximum limit to 100
-          include_total_count: true   # include total count by default
+          include_total_count: true,  # include total count by default
+          primary_key: :uuid          # sets the primary_key to uuid for calculate total_count
```

Comment:
Since we only use this to calculate the total count, can we rename this to `total_count_primary_key_field`?